R/shinyui.R

```diff
@@ -82,6 +63,27 @@ renderPage <- function(ui, showcase=0, testMode=FALSE) {
   enc2utf8(paste(collapse = "\n", html))
 }
 
+jqueryDependency <- function() {
+  version <- getOption("shiny.jquery.version", 3)
+  if (version == 3) {
+    return(htmlDependency(
+      "jquery", version_jquery,
+      src = "www/shared",
```

Comment:
Do we want to do something like:

```R
src = c(file=filepath, href=url)
```

There are some cases where `file` is used and others where `href` is used, but I can't remember the details -- we should find that out before we make the change.

Reply:
Yea, good point. By including that, I was able to confirm the `href` is preferred when rendering via `shinyApp()` (BTW, `bootstrapLib()` also defines both `file` and `href`)